src/java.base/share/classes/jdk/internal/access/JavaLangAccess.java

```diff
@@ -345,8 +349,10 @@ public interface JavaLangAccess {
     String newStringUTF8NoRepl(byte[] bytes, int off, int len);
 
     /**
-     * Get the char at index in a byte[] in internal UTF-16 representation,
-     * with no bounds checks.
+     * Get the {@code char} at {@code index} in a {@code byte[]} in internal
+     * UTF-16 representation.
+     * <p>
+     * <b>WARNING: This method does not perform any bound checks.</b>
```

Comment:
Maybe not this PR but this method is begging to be renamed. It's really hard to audit usage when the method is accessible to faraway code via shared secrets. So maybe not this PR but I think we need an issue in JBS to track renaming, I think Stuart suggested prefixing with "unchecked".

Reply:
As mentioned by Alan and Jaikiran, at some point it is advisable to prefix the names with "unchecked".

For the "noRepl" names, perhaps the mouthful but clearer "noReplacementChar" might help.

Reply:
@jaikiran, @AlanBateman, implemented `unchecked` method prefix in 4d237efa053.